ee/app/license/server/bundles.ts

```diff
@@ -14,10 +14,10 @@ const bundles: IBundle = {
 	],
 };
 
-const getBundleFromModule = (moduleName: string): string|undefined => {
+export const getBundleFromModule = (moduleName: string): string => {
```

Comment:
Isn't this misleading since an empty string isn't a module and may be checked somewhere as `!= null`?

Reply:
it will still work if you compare with `!= null`, so no issue with this. it would break though if there was a code comparing `!== ''`. but it was not the case.

but the only reason for this change was actually to be able to use in this [.map](https://github.com/RocketChat/Rocket.Chat/pull/18093/files#diff-8d82562ac20c763d0b8975b4c46a630dR83) .. I didn't found other solution for this, suggestions are welcome. this is what TS complains if keep it as `string|undefined`:
![image](https://user-images.githubusercontent.com/8591547/87799860-1bdbf800-c824-11ea-987c-5eff27862de0.png)

Reply:
You can change the forEach to
```javascript
.forEach((m) => m && this.tags.add(m));
```
and prevent from add an empty string as a tag as well

Reply:
not that beauty but it works.. I'd like it to be clever and work with an additional `filter`:
```javascript
			license.modules
				.filter(isBundle)
				.map(getBundleFromModule)
				.filter((tag) => !!tag)
				.forEach(this.tags.add, this.tags);
```

but I'll revert this 👍